.circleci/cimodel/data/simple/non_caffe2_definitions.py

```diff
@@ -0,0 +1,62 @@
+"""
+PyTorch builds without CAFFE2 options
+"""
+
+import cimodel.lib.miniutils as miniutils
+import cimodel.data.simple.util.branch_filters
+from cimodel.data.simple.util.docker_constants import (
+    DOCKER_IMAGE_BASIC,
+    DOCKER_REQUIREMENT_BASE,
+)
+
+
+class NonCaffe2Job:
```

Comment:
How about `PyTorchPure` build job? Also, I think it would be better to just add one more experimental node (say `no-caffe2`) to
https://github.com/pytorch/pytorch/blob/6ea89166bd685fbae59da6dba5bf1d11d55e4ea8/.circleci/cimodel/data/pytorch_build_data.py#L146
than create a new config altogether